source/administration/object-management/data-compression.rst

```diff
@@ -48,7 +48,7 @@ You can control which objects are compressed by specifying the desired file exte
 Excluded File Types
 ~~~~~~~~~~~~~~~~~~~
 
-Some data cannot be effectively compressed, such as video or already compressed data.
+Some data cannot be effectively compressed, such as video, already compressed data, or very small files.
```

Comment:
```suggestion
Some data cannot be effectively compressed.
For example: video, already compressed data, or very small files.
```

A suggestion to consider to avoid a dizzying amount of commas.

Reply:
oh yes, that is nicer 👍🏻